### PR TITLE
[JENKINS-65161] remove commons-digester2 from core and upgrade plugins to commons-digester3

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

**Comment by olamy** [comment]
see https://github.com/jenkinsci/jenkins/pull/5320

**Comment by olamy** [comment]
@mig42 ping as you are the main maintainer

**Comment by mig42** [comment]
Sure, we'll get this merged and released ASAP 😊

File: pom.xml
Comment (rsandell) [inline pom.xml]: It is merged now

**Review by rsandell — DISMISSED** [review]

File: pom.xml
Comment (olamy) [reply pom.xml]: fixed thanks!

File: src/main/java/com/codicesoftware/plasticscm/plugins/mergebot/jenkins/PlasticChangeLogParser.java
Comment (MRamonLeon) [inline src/main/java/com/codicesoftware/plasticscm/plugins/mergebot/jenkins/PlasticChangeLogParser.java:7]: The important line

**Review by MRamonLeon — APPROVED** [review]

**Review by alecharp — APPROVED** [review]

**Comment by alecharp** [comment]
@mig42 any blockers to get this PR merge and cut a release of the plugin? Thanks.

**Comment by olamy** [comment]
@mig42 ping :)

**Review by mig42 — APPROVED** [review]
Looks good to me! I'll merge the PR once I verified it works in my environment.

**Pull request merged**
